packages/next/test/fixtures/00-app-dir-segment-options/next.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  experimental: {
+    appDir: true,
```

Comment:
You can remove `appDir: true` since its no longer needed https://nextjs.org/blog/next-13-4

I'm not sure but I think `runtime` can be removed here too

Reply:
I'll check, thank you

Reply:
Removed